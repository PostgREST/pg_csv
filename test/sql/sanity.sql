SELECT csv_agg(x) AS body
FROM   projects x;

-- proves that https://github.com/PostgREST/postgrest/issues/1371#issuecomment-519248984 is solved
select csv_agg(x) from nasty x;
