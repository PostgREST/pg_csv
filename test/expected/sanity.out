SELECT csv_agg(x) AS body
FROM   projects x;
             body              
-------------------------------
 id,name,client_id            +
 1,Windows 7,1                +
 2,"has,comma",1              +
 ,,                           +
 4,OSX,2                      +
 ,"has""quote",               +
 5,"has,comma and ""quote""",7+
 6,"has                       +
  LF",7                       +
 7,"has \r CR",8              +
 8,"has \r                    +
  CRLF""",8
(1 row)

-- proves that https://github.com/PostgREST/postgrest/issues/1371#issuecomment-519248984 is solved
select csv_agg(x) from nasty x;
        csv_agg         
------------------------
 "unusual"",names",text+
 1,test
(1 row)

